app/src/main/java/fi/sabriina/urbanhuikka/MainActivity.kt

```diff
@@ -248,6 +261,7 @@ class MainActivity : AppCompatActivity(), OnCardSwipeListener {
                 }
             }
             splashScreenManager.showSplashScreen(currentPlayer.name, currentPlayerPicture,"Sait ${currentCard!!.points} pistettä!", drawableDrink)
+            splashScreenManager.showSplashScreen(assistingPlayer.name,assistingPlayerPicture, "Sait ${currentCard!!.points} pistettä!", drawableDrink)
```

Comment:
Assisting player drinks with every card skip, even though they didn't participate in the current card.